react/react1/week2/todo/src/TodoList.js

```diff
@@ -0,0 +1,84 @@
+import { useState } from "react";
+import React from "react";
+import { TodoItem } from "./TodoItem";
+
+const todos = [
+  {
+    id: 1,
+    description: "Get out of bed",
+  },
+  {
+    id: 2,
+    description: "Brush teeth",
+  },
+  {
+    id: 3,
+    description: "Eat breakfast",
+  },
+];
+const addTodos = [
+  { description: "dress well" },
+  { description: "go to school" },
+  { description: "study hard" },
+  { description: "Eat lunch" },
+  { description: "do sports" },
+  { description: "back home" },
+];
+// main function
+function TodoList() {
+  const [usetodo, setUsetodo] = useState(todos);
```

Comment:
Avoid naming variables in React with "use" since it is easy to confuse them as hooks. Rename the initial value instead! 
```suggestion
  const [todos, setTodos] = useState(initialTodos);
```